src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -114,7 +115,30 @@ function repeatableInputToObj(container_name) {
          */
         function bpFieldInitRepeatableElement(element) {
 
-            var field_name = element.attr('name');
+            let field_name = element.attr('name');
+            let form = element.closest('form');
+            let form_events = jQuery._data(form[0], 'events');
+
+            // create a new event handler that will parse the repeatable values to the hidden inputs
+            // so they can be submited along with form when requesting some ajax endpoint
+            // we check that the event is not registered twice
+            if (!('backpack_field' in form_events) || !Object.values(form_events.backpack_field).find(e => e.namespace === 'parse_value')) {
```

Comment:
@pxpm, when does it enter the second part of the condition?
I wasn't able to test that part.

Reply:
`backpack_field` is also the event namespace for `backpack_field.deleted`, so to go to the second condition you have to have a ckeditor that listens for the `backpack_field.deleted`.

Also enters the second condition everytime `backpack_field` exists in the `events` where it checks if we have any `parse_value` event already registed. 

In dummy example, where you have 5 or 6 `repeatable` fields, it will attempt to register 5,6 times the event, in the second attempt it will enter the condition and fail, so don't register the event that many times, only the first time.

Let me know,
Pedro